requirements.txt

```diff
@@ -6,3 +6,5 @@ pandas==2.2.2
 sentence-transformers==3.0.1
 tiktoken==0.7.0
 tqdm==4.66.4
+qwen_vl_utils==0.0.8
+torchvision==0.19.0
```

Comment:
Does `torchvision` need to be a requirement? Seems its only used in the example for loading a dataset

Reply:
was needed by qwen_2. Removed